src/org/folio/rest_v2/Main.groovy

```diff
@@ -80,15 +90,25 @@ class Main extends Okapi {
         }
     }
 
-    void update(Map<String, OkapiTenant> tenants){
+    void update(Map<String, OkapiTenant> tenants) {
         tenants.each { tenantId, tenant ->
             tenantInstall(tenant, tenant.modules.generateInstallJsonFromIds(['okapi'], 'enable'))
             tenantInstall(tenant, tenant.modules.installJson, 900000)
-            configureTenant(tenant)
+            if (tenant.getAdminUser()) {
+                loginUser(tenant)
+                permissions.purgeDeprecatedPermissions(tenant)
+                permissions.refreshAdminPermissions(tenant, tenant.getAdminUser())
+//                configureTenant(tenant)
```

Comment:
I’ll point it out just in case this shouldn’t be here 👀

Reply:
It should but need some adjustments